engine/access/rest/websockets/controller_test.go

```diff
@@ -483,8 +483,10 @@ func (s *WsControllerSuite) TestListSubscriptions() {
 
 		id := uuid.New()
 		topic := dp.BlocksTopic
+		arguments := models.Arguments{}
```

Comment:
can you add some value into the arguments map to ensure we're testing that the same set is returned in the asserts. Otherwise we could miss a bug where an empty argument set is passed instead